content/guides/ethereum/tutorials/track_trx.md

```diff
@@ -31,7 +31,7 @@ then open ({{< external-link href="http://localhost:3000/">}})
 
 ## 3. Add the dfuse Client Library
 
-The simplest way to get started with dfuse and JavaScript/TypeScript development is to use the dfuse JS client library.
+The simplest way to get started with dfuse and JavaScript/TypeScript development is to use the dfuse JS Client Library.
```

Comment:
```suggestion
The simplest way to get started with dfuse and JavaScript/TypeScript development is to use the dfuse JS client library.
```

Reply:
Damn, it's on your fork, so I cannot auto-add the suggestions :$

Going to merge and make the changes manually then.